README.md

```diff
@@ -20,6 +20,13 @@ The module creates the following resources:
 - Review this module with your Cloud Security team to ensure that it meets your security requirements.
 - Finally, after the Terraform module has been applied, you will need to make sure that the Target Groups heatlth checks are passing. As the NLB does not have security groups, you will need to make sure that the NLB is able to reach the Kafka brokers by allowing the subnet CIDR blocks in the security groups of the Kafka cluster.
 
+Export the following environment variables:
```

Comment:
Like the other PR, we should clarify that these aren’t required unless they need to override the default. Otherwise, looks good to me

Reply:
Good point! Just updated the message, let me know what you think!